### DISCUSSION TITLE
Cancellable tasks

### DISCUSSION THREAD

**Discussion by igor-yusupov** [fzyzcjy/flutter_rust_bridge#1689] [Q&A]
I see the topic: https://cjycode.com/flutter_rust_bridge/v1/feature/cancelable_task.html
I also see the comment: https://github.com/fzyzcjy/flutter_rust_bridge/pull/333#issuecomment-1037079219

But nevertheless it's not quite clear to me if it's now possible to call canceling the execution of a function?

**Comment by fzyzcjy** [comment]
The new doc is here: https://cjycode.com/flutter_rust_bridge/guides/how-to/cancel. Hope it is a bit clearer!
